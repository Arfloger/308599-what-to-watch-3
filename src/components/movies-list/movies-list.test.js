import React from 'react';
import renderer from 'react-test-renderer';
import MoviesList from './movies-list.jsx';

it(`<MoviesList/> should render with films`, () => {

  const tree = renderer
  .create(
      <MoviesList
        films={[]}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
