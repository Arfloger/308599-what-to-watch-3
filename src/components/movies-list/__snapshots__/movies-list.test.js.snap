// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviesList/> should render with films 1`] = `
<div
  className="catalog__movies-list"
/>
`;
